cms/static/js/features_jsx/studio/index.jsx

```diff
@@ -1,104 +1,131 @@
-import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import React from 'react';
 import ReactDOM from 'react-dom';
 
-class CourseOrLibraryListing extends React.Component {
+function CourseOrLibraryListing(props) {
+  /* global gettext */
+  /* eslint react/no-array-index-key: 0 */
```

Comment:
Yeah, honestly I kind of hate `react/no-array-index-key` and was really tempted to disable it globally in `eslint-config-edx`. I think it's okay to disable inline for now, but I'd move these two lines up to the very top of the file, the same way we do with pylint ignore rules.